### PR TITLE
update twitter link

### PR DESCRIPTION
**Pull request by imtuyethan**
## Describe Your Changes
Update website's Twitter link

### CODE REVIEW

**Review by urmauur — APPROVED** [review]

**Comment by urmauur** [comment]
LGTM

**Review by louis-jan — APPROVED** [review]
LGTM

File: plugins/inference-plugin/nitro/mac-arm64/ggml-metal.metal
Comment (louis-jan) [inline plugins/inference-plugin/nitro/mac-arm64/ggml-metal.metal:1]: Please discard these changes

**Review by 0xSage — CHANGES_REQUESTED** [review]
Pls discard changes in /nitro

**Review by louis-jan — APPROVED** [review]
LGTM

**Review by 0xSage — APPROVED** [review]

**Pull request closed without merging**